### PR TITLE
US115894: part 3 call sort in the start

### PR DESCRIPTION
**Pull request by msanchezal**

### CODE REVIEW

File: src/cardstack/al-cardstack-view.ts [hunk 1]
```diff
@@ -238,21 +242,24 @@ export abstract class AlCardstackView< EntityType=any,
      *  Returning `true` indicates that the current list of items needs to be flushed and data retrieval should start from scratch.
      */
     public applySortBy( descriptor:AlCardstackPropertyDescriptor, order:string = "DESC" ):boolean {
+        this.sortingBy = descriptor;
+        this.sortOrder = order;
```
Comment (scottarver) [inline src/cardstack/al-cardstack-view.ts:246]: apply toLowerCase()

File: src/cardstack/al-cardstack-view.ts [hunk 2]
```diff
@@ -38,7 +38,7 @@ export abstract class AlCardstackView< EntityType=any,
     public textFilter:      string|RegExp|null                                         =   null;       //  Regular expression to filter results with (deprecated?)
     public groupingBy:      AlCardstackPropertyDescriptor|null                  =   null;       //  Grouping property
     public sortingBy:       AlCardstackPropertyDescriptor|null                  =   null;       //  Sortation property
-    public sortOrder:       string                                              =   "ASC";      //  Sortation direction, either "ASC" or "DESC".  Yes, "sortation" is a real word ;-)
+    public sortOrder:       string                                              =   "asc";      //  Sortation direction, either "asc" or "desc".  Yes, "sortation" is a real word ;-)
```
Comment (scottarver) [inline src/cardstack/al-cardstack-view.ts:41]: instead of string, mark as 'asc'|'desc'

File: src/cardstack/al-cardstack-view.ts [hunk 3]
```diff
@@ -238,21 +242,24 @@ export abstract class AlCardstackView< EntityType=any,
      *  Returning `true` indicates that the current list of items needs to be flushed and data retrieval should start from scratch.
      */
     public applySortBy( descriptor:AlCardstackPropertyDescriptor, order:string = "DESC" ):boolean {
```
Comment (scottarver) [inline src/cardstack/al-cardstack-view.ts:244]:
```suggestion
    public applySortBy( descriptor:AlCardstackPropertyDescriptor, order:string = "dsc" ):boolean {
```

File: src/cardstack/al-cardstack-view.ts (hunk 1 above)
Comment (scottarver) [inline src/cardstack/al-cardstack-view.ts:246]:
```suggestion
        this.sortOrder = order.toLowerCase();
```

File: src/cardstack/al-cardstack-view.ts (hunk 2 above)
Comment (scottarver) [inline src/cardstack/al-cardstack-view.ts:41]:
```suggestion
    public sortOrder:       'asc'|'desc'                                         =   "asc";      //  Sortation direction, either "asc" or "desc".  Yes, "sortation" is a real word ;-)
```

Comment (msanchezal) [reply src/cardstack/al-cardstack-view.ts:41]: i like this way instead of lowercase

**Pull request merged**